ui/desktop/package.json

```diff
@@ -28,7 +28,7 @@
     "format:js": "prettier --write '{app,config,stories,tests,electron-app/src}/**/*.js' *.js",
     "format:sass": "prettier --write app/**/*.scss",
     "lint:js:fix": "eslint . --fix",
-    "start": "ember serve",
+    "start": "ember serve --port 3200",
```

Comment:
@randallmorey @gsusmi be aware of this change. I add this to could work in the desktop with the browser without any port issues with the admin.

Reply:
I'd vote to leave port configurations out of our start scripts, mainly because we standardize on start scripts in every available `package.json` and all of us have our port range preferences. I personally use `yarn start -p <port>` when launching multiple clients/addons.

Reply:
Fair enough. I do agree that should be more developer preference, thanks for the tip on the `--port` flag for the yarn script :wink: